config/settings.py

```diff
@@ -17,17 +17,17 @@
     CI=(bool, False),
     AMY_DEBUG=(bool, True),
     AMY_SITE_ID=(int, 2),
-    AMY_ALLOWED_HOSTS=(list, ["amy.carpentries.org"]),
-    AMY_DATABASE_HOST=(str, "localhost"),
+    AMY_ALLOWED_HOSTS=(list, ["amy.charminghat.nl"]),
+    AMY_DATABASE_HOST=(str, "database"),
     AMY_DATABASE_PORT=(int, 5432),
     AMY_DATABASE_NAME=(str, "amy"),
     AMY_DATABASE_USER=(str, "amy"),
     AMY_DATABASE_PASSWORD=(str, "amypostgresql"),
     AMY_RECAPTCHA_PUBLIC_KEY=(str, "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"),
     AMY_RECAPTCHA_PRIVATE_KEY=(str, "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"),
-    AMY_SOCIAL_AUTH_GITHUB_KEY=(str, ""),
-    AMY_SOCIAL_AUTH_GITHUB_SECRET=(str, ""),
-    AMY_GITHUB_API_TOKEN=(str, ""),
+    AMY_SOCIAL_AUTH_GITHUB_KEY=(str, "918dec4feb5f6568b544"),
+    AMY_SOCIAL_AUTH_GITHUB_SECRET=(str, "0dd0074b2a80c104427a7cf5e647567ad7b0e2e1"),
+    AMY_GITHUB_API_TOKEN=(str, "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"),
```

Comment:
Hey @thomasve-DANS / @tvanerven, you've revealed some secrets in your fork. For security reasons you should re-create these and store them somewhere private. You can put environment variables in a file named `.env` in your AMY root directory - AMY is configured to [read `.env` on startup](https://github.com/carpentries/amy/blob/develop/config/settings.py#L51), and [Git will ignore it](https://github.com/carpentries/amy/blob/develop/.gitignore#L49).

Reply:
Yeap, spotted. I've regenerated the keys, and moved it towards .env file. Thanks for the tag : ) And I'll make sure these things don't go through to the upstream fork.